app/main.py

```
@@ -1,9 +1,12 @@
+from datetime import datetime
+from app.saver.db_saver import save_statistics
 from app.config.database import get_engine
 from app.data.loader import *
```

Ruff (0.12.2)

`from app.data.loader import *` used; unable to detect undefined names

(F403)